### ISSUE TITLE
Datagram buffer size issues

### ISSUE THREAD

**Issue by MOZGIII** [MOZGIII/xwt#101]
Something's off with datagram buffer sizes in wasm; probably a simple fix, but first an investigation of what and why is happening in needed. And a test case.

**Comment by aecsocket** [comment]
Reproducible example: https://github.com/aecsocket/aeronet/tree/a416cf110f5bd1e627e93124a654f90a977a2681 (this specific commit)

Instructions to reproduce:
* Start the server
  * `cargo run -p aeronet_webtransport --example echo_server --features "bevy aeronet/bevy-tokio-rt"`
* Close your existing browser windows
* Restart your browser with these flags
```
brave \
--webtransport-developer-mode \
--ignore-certificate-errors-spki-list=x3S9HPqXZTYoR2tOQMmVG2GiZDPyyksnWdF9I9Ko/xY=
```
* Run the WASM client (install wasm-server-runner first)
  * `cargo run -p aeronet_webtransport --example echo_client --features "bevy dangerous-configuration" --target wasm32-unknown-unknown`
* Go to the localhost link that the above command gives you, and type in `https://[::1]:25565` and connect

**Comment by MOZGIII** [comment]
This should be fixed by #103. Please check and close if it works.
